apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -66,9 +66,10 @@
 #ifdef CONFIG_FS_SMARTFS
 #if defined(CONFIG_SIDK_S5JT200_AUTOMOUNT_USERFS)
 #define TMP_MOUNT_DEV_DIR CONFIG_SIDK_S5JT200_AUTOMOUNT_USERFS_DEVNAME
-
 #elif defined(CONFIG_ARTIK05X_AUTOMOUNT_USERFS)
 #define TMP_MOUNT_DEV_DIR CONFIG_ARTIK05X_AUTOMOUNT_USERFS_DEVNAME
+#elif defined(CONFIG_ARCH_BOARD_LM3S6965EK)
+#define TMP_MOUNT_DEV_DIR "/dev/smart0p0"
```

Comment:
Is it fixed always as smart0p0?

Reply:
It is created using minor no. and part no. and they are not configurable. So I hard coded it here.
@manoharahk kindly share your opinion about this change

Reply:
I believe, it's better to provide this interface through defconfig as it remains a common place to refer for used device name and minor no for all kinds of file systems

Reply:
@lokeshbv @pradeepns85 I agree Pradeep's suggestion. We had better one file for fs mount point interface. Now, it is defined at each location with different types.